inspirehep/modules/records/jsonschemas/records/elements/rank.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "enum": [
+      "STAFF"
+      "SENIOR"
+      "JUNIOR"
+      "VISITOR"
+      "POSTDOC"
+      "PHD"
+      "MASTER"
+      "BACHELOR"
+      "UNDERGRADUATE"
+      "STUDENT",
+      "OTHER"
```

Comment:
Does that means that we are moving away from the current `enum` used on legacy? I am ok with that, just raising the point as this has implications e.g. in the author form etc.